sdk/translation/azure-ai-translation-text/azure/ai/translation/text/_patch.py

```diff
@@ -79,6 +78,15 @@ def get_translation_endpoint(endpoint, api_version):
 
     return translator_endpoint
 
+def is_cognitive_services_scope(audience: str):
+    if not audience:
+        return False
```

Comment:
when would audience be falsey? It looks like default it if it's not present

Reply:
It was just a copy-paste, removed.